lang/en/tool_excimer.php

```diff
@@ -142,6 +144,7 @@
 $string['field_hostname'] = 'Host name';
 $string['field_useragent'] = 'User agent';
 $string['field_versionhash'] = 'Version Hash';
+$string['field_course'] = 'Course';
```

Comment:
This and 'Courses' I expect would already exist in core, and can be reused

Reply:
We can't reuse them since the excimer plugin looks for `get_string('field_' . $fieldname)`

Reply:
https://github.com/catalyst/moodle-tool_excimer/blob/5257c205ba836337b01b9f5f2780efea73332bcc/classes/profile_table.php#L90